timesketch/lib/analyzers/contrib/hashlookup_analyzer_test.py

```diff
@@ -0,0 +1,42 @@
+"""Test for Hashlookup"""
```

Comment:
You're not testing the analyzer here, you're testing the `hash_match` function :)

Have a look at https://github.com/google/timesketch/blob/master/timesketch/lib/analyzers/yetiindicators_test.py to see how we do it; but you need to use your actual analyzer object and probably also some mocks on the request response object.. Happy to provide more guidance on how to do this.